File: Arreglos/310826/SistAlumnosDin.cpp
#include <iostream>
#include <string>

int main() {
    std::string alumnos[5][3];

    std::cout << "--- INGRESO DE DATOS (C++) ---\n";
    for (int i = 0; i < 5; i++) {
        std::cout << "Alumno " << (i + 1) << ":\n";
        std::cout << "  Nombre: ";
        std::cin >> alumnos[i][0];
        std::cout << "  Salon: ";
        std::cin >> alumnos[i][1];
        std::cout << "  Promedio: ";
        std::cin >> alumnos[i][2];
    }

    std::cout << "\n--- DATOS REGISTRADOS ---\n";
    for (int i = 0; i < 5; i++) {
        std::cout << "Alumno: " << alumnos[i][0] 
                  << " | Salon: " << alumnos[i][1] 
                  << " | Promedio: " << alumnos[i][2] << std::endl;
    }

    return 0;
}

File: Arreglos/310826/SistAlumnosDin.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

const alumnos = [];
let i = 0;

console.log("--- INGRESO DE DATOS (JavaScript) ---");

function pedirDatos() {
    if (i < 5) {
        console.log(`\nAlumno ${i + 1}:`);
        rl.question("  Nombre: ", (nombre) => {
            rl.question("  Salon: ", (salon) => {
                rl.question("  Promedio: ", (promedio) => {
                    alumnos.push([nombre, salon, promedio]);
                    i++;
                    pedirDatos();
                });
            });
        });
    } else {
        console.log("\n--- DATOS REGISTRADOS ---");
        alumnos.forEach(alumno => {
            console.log(`Alumno: ${alumno[0]} | Salon: ${alumno[1]} | Promedio: ${alumno[2]}`);
        });
        rl.close();
    }
}

pedirDatos();